flink-table-store-core/src/main/java/org/apache/flink/table/store/file/manifest/ManifestCommittable.java

```diff
@@ -27,26 +27,26 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
+import java.util.UUID;
 
 /** Manifest commit message. */
 public class ManifestCommittable {
 
+    private final String uuid;
     private final Map<BinaryRowData, Map<Integer, List<SstFileMeta>>> newFiles;
-
     private final Map<BinaryRowData, Map<Integer, List<SstFileMeta>>> compactBefore;
-
     private final Map<BinaryRowData, Map<Integer, List<SstFileMeta>>> compactAfter;
 
     public ManifestCommittable() {
-        this.newFiles = new HashMap<>();
-        this.compactBefore = new HashMap<>();
-        this.compactAfter = new HashMap<>();
+        this(UUID.randomUUID().toString(), new HashMap<>(), new HashMap<>(), new HashMap<>());
     }
 
     public ManifestCommittable(
+            String uuid,
```

Comment:
Can we use checkpoint id here? Just like https://github.com/apache/flink-table-store/pull/22